main.py

```diff
@@ -0,0 +1,5 @@
+import MediaPlayer
+import os
+songs_list=MediaPlayer.load_dir_to_dictionary()
```

Comment:
In python it's idiomatic to wrap the main functionality of a program in a function called main, and then invoke it bellow a line that looks like this:
```
if __name__ == "__main__":
    main()
```
We can talk next time about why